### ISSUE TITLE
Add warning message if workers are down

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#489]
**Is your feature request related to a problem? Please describe.**
We want to move a lot of workload over to workers, but if workers are down we would silently fail to do a lot of crucial functions

**Describe the solution you'd like**
A big red warning message somewhere (maybe in the top right in the dropdown menu?) that explains workers are down

See #455 for more context

**Comment by mariusandra** [comment]
Hi!

I did the following changes (in PR #494)

1. When you start PostHog without REDIS_URL defined, you'll see a warning in the console. We should replace this with an error in 1.1.0.
![image](https://user-images.githubusercontent.com/53387/78751177-04ae8400-7972-11ea-88bd-2e32ed7b6f73.png)

2. When either redis is not configured (pre 1.1.0) or when the worker is offline (all versions), you'll see this error on the top of the app (the "posthog up to date" message should be to the left of it... but the update server gives `500` now)
![image](https://user-images.githubusercontent.com/53387/78765620-88736b00-7988-11ea-886f-11e5e360dc55.png)

3. The worker sends a heartbeat every 60 seconds. The frontend checks to see how long it has been since the last heartbeat. If it was more than 90 seconds, we'll show this orange message instead:
![image](https://user-images.githubusercontent.com/53387/78766287-60383c00-7989-11ea-9c44-c8e14c1e81d2.png)

4. When you click either the red or the orange message, we show this modal that opens [this URL](https://docs.posthog.com/#/upgrading-posthog?id=upgrading-from-before-1011).
![image](https://user-images.githubusercontent.com/53387/78766354-73e3a280-7989-11ea-82a1-889ef27eb957.png)

**Comment by jamesefhawkins** [comment]
* Let's make it <!> configuration error instead of !! Worker OFFLINE !!
* The modal is good - ie giving more explanation on click. Could we reword this as: CONFIGURATION ISSUE, then a few sentences that explain to someone non-technical the implications THEN "details" for an engineer to understand what the actual problem is. We should probably put why we made this change too, as it's a good thing but comes across as annoying if you just see "urgh I need to install background workers somehow and I don't get why"

**Comment by mariusandra** [comment]
Hey, sure!

How about this (already in the PR):

![image](https://user-images.githubusercontent.com/53387/78769420-a42d4000-798d-11ea-82af-9a857a02b563.png)

![image](https://user-images.githubusercontent.com/53387/78769392-9b3c6e80-798d-11ea-9793-16c0035fa76b.png)

Also, in case everything is fine with the workers, we show no message at all.
